### DISCUSSION TITLE
Configuring ltsp client to login to Windows through RDP

### DISCUSSION THREAD

**Discussion by giesekow** [ltsp/ltsp#397] [General]
I wanted to add a menu item for booting to windows through `xfreerdp`. This was achievable in the old version of LTSP through the `lts.conf` file by
SCREEN_07=xfreerdp
RDP_SERVER=server_ip

Is it possible to achieve similar result in the new LTSP and how do we do this?

**Comment by alkisg** [comment]
You can create custom sessions like this:
https://askubuntu.com/questions/77191/how-can-i-use-lightdm-for-user-defined-sessions

Then users can select the session they want in the login screen.

**Comment by alkisg** [comment]
Converting to a discussion, as it's not an LTSP bug.
